code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -64,11 +64,13 @@
 	..()
 	lash = new
 	lash.Grant(H)
+	H.mind.offstation_role = TRUE
 
 /datum/species/unathi/on_species_loss(mob/living/carbon/human/H)
 	..()
 	if(lash)
 		lash.Remove(H)
+	H.mind.offstation_role = FALSE
```

Comment:
This isn't neccesary iirc. If an Ash Walkers species changes for whatever, cling sting, prides mirror, etc. They shouldn't become eligible to be an objective target.

Reply:
Also mirroring S34N, this will cause all unathis everywhere to be ineligible for being an objective.